GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
getGVRContext() needs javadoc describing when it makes sense to be called; even better, in addition to that it should throw an IllegalStateException if it is called before setScript().